app/src/main/java/com/mealsmadeeasy/data/FakeMealStore.kt

```diff
@@ -143,4 +143,25 @@ class FakeMealStore : MealStore {
 
         return Single.just(listOf(tacos, iceCream))
     }
+
+    override fun findMealById(id: String): Meal {
+        val tacos = Meal(
+                id = "tacos", name = "Tacos",
+                description = """
+                    It's the best food product.
+                """.trimIndent(),
+                thumbnailUrl = "https://www.forksoverknives.com/wp-content/uploads/lentiltacos-6553WP-edit.jpg")
+
+        val iceCream = Meal(
+                id = "ice_cream", name = "Chocolate ice cream",
+                description = """
+                    It's chocolate ice cream. Yup.
+                """.trimIndent(),
+                thumbnailUrl = "https://chocolatecoveredkatie.com/wp-content/uploads/2017/02/nice-cream.jpg"
+        )
```

Comment:
It may help to have a list of all the meals we make in `init` so we don't have to explicitly put these here. It's just a dummy implementation, so it doesn't have to be perfect, I just think it'd be something to address in case we call this with "bagels" or something since it's technically a valid id for this meal store.